src/Console/DuskCommand.php

```diff
@@ -68,9 +69,15 @@ public function handle()
                 $this->output->writeln('Warning: '.$e->getMessage());
             }
 
-            return $process->run(function ($type, $line) {
-                $this->output->write($line);
-            });
+            try {
+                return $process->run(function ($type, $line) {
+                    $this->output->write($line);
+                });
+            } catch (ProcessSignaledException $e) {
+                if ($e->getSignal() !== 2) {
```

Comment:
Can we replace the `2` by a predefined constant?

Reply:
Good one, have replaced it with `SIGINT`, global constant provided by the PCNTL extension: https://www.php.net/manual/en/pcntl.constants.php